internal/otelcollector/config/log/agent/testdata/config.yaml

```diff
@@ -92,6 +92,26 @@ receivers:
               type: severity_parser
               if: attributes.level != nil
               parse_from: attributes.level
+            - id: trace-parent-parser
+              type: regex_parser
+              if: attributes.trace_id == nil
```

Comment:
Shouldn't have the second parser the same if condition?

Reply:
solved via router operator